project1/src/components/Buttons/styles/DeleteButtonStyle.ts

```diff
@@ -0,0 +1,30 @@
+import { createStyles } from "@material-ui/core/styles";
+
+export const styles = () =>
+  createStyles({
+    root: {
+      float: "right",
```

Comment:
same as favorite button, please remove remove css properties that makes the button not reusable